tests/integration/test_record.py

```diff
@@ -139,3 +180,8 @@ def test_record_can_be_deleted(app, record_not_yet_deleted):
         db.session.commit()
 
         assert client.get('/api/literature/333').status_code == 410
+
+
+def test_record_can_be_immediately_es_updated(app, record_not_es_refreshed):
+    with app.test_client() as client:
+        assert client.get('/api/literature/444').status_code == 200
```

Comment:
There is a difference: the refresh is really needed if one wants to see what ES has actually indexed. Instead this particular test is a regression test to demonstrate that visiting a record being indexed in this very moment doesn't lead to a 500.